packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/template.html

```diff
@@ -131,13 +131,13 @@
         </oui-field>
 
         <!--User credential overview information-->
-        <pci-project-storages-user-credential-banner
+        <storages-components-user-credential-banner
```

Comment:
```suggestion
        <pci-projects-project-storages-components-user-credential-banner
```

Reply:
Here, we are using the same component used for Cold-archive container creation (new user & linked user) and the same component for Object storage container creation (linked user).

The component name has already been validated in other review, changing its name is outside of the the scope of this PR.

Reply:
The PR which was validated before maybe a mistake. If you feel renaming the component is out of the scope of this PR, can you create a refactor task to handle the rename of the component?

Reply:
I've created the refactor task : MANAGER-11178